src/bin/cargo-miri.rs

```diff
@@ -369,13 +424,20 @@ path = "lib.rs"
         None => true,
         Some(target) => target == rustc_version::version_meta().unwrap().host,
     };
-    let sysroot = if is_host { dir.join("HOST") } else { PathBuf::from(dir) };
+    let sysroot = if is_host {
+        dir.join("HOST")
+    } else {
+        PathBuf::from(dir)
+    };
     std::env::set_var("MIRI_SYSROOT", &sysroot); // pass the env var to the processes we spawn, which will turn it into "--sysroot" flags
     if print_sysroot {
         // Print just the sysroot and nothing else; this way we do not need any escaping.
         println!("{}", sysroot.display());
     } else if !ask_user {
-        println!("A libstd for Miri is now available in `{}`.", sysroot.display());
+        println!(
+            "A libstd for Miri is now available in `{}`.",
+            sysroot.display()
+        );
```

Comment:
Another linebreak of a line that deosn't seem like it is too long.

Reply:
I can override the rustfmt default line length to use 100 instead of 80

Reply:
Please do.